xm_drupal_manual.module

```diff
@@ -73,3 +73,13 @@ function xm_drupal_manual_form_alter(&$form, \Drupal\Core\Form\FormStateInterfac
     unset($form['book']);
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function xm_drupal_manual_preprocess_book_export_html(&$variables)
+{
+    // path to module
+    $path = Drupal::service('extension.list.module')->getPath('xm_drupal_manual');
```

Comment:
As we don't reuse the `$path` variable, we don't need it. Please remove it and save memory doing that.